setup.py

```diff
@@ -35,10 +41,6 @@ def get_install_requires():
             continue
         # add line to requirements
         requirements.append(line.replace('\n', ''))
-    # add py2-ipaddress if python2
```

Comment:
not sure this block of code can be removed, it may still be required for non-wheel packages.